File: aoj/itp1/topic1/1_d_watch.cc
#include <iostream>

int main() {
  int total_seconds;
  std::cin >> total_seconds;

  int hours = total_seconds / 3600;
  int minutes = total_seconds / 60 - hours * 60;
  int seconds = total_seconds % 60;
  std::cout << hours << ":" << minutes << ":" << seconds << std::endl;
}

File: aoj/itp1/topic1/1_d_watch.java
import java.util.Scanner;

class Main {
  public static void main(String[] args) {
    Scanner sc = new Scanner(System.in);
    int s = sc.nextInt();
    int hours = s / 3600;
    int minutes = s % 3600 / 60;
    int seconds = s % 60;
    System.out.println(hours + ":" + minutes + ":" + seconds);
  }
}
